samza-api/src/main/java/org/apache/samza/system/SystemAdmin.java

```diff
@@ -114,4 +114,4 @@ default boolean createStream(StreamSpec streamSpec) {
   default void validateStream(StreamSpec streamSpec) throws StreamValidationException {
     throw new UnsupportedOperationException();
   }
-}
+}
```

Comment:
nit: can we try to get rid of these minor changes that are seemingly unrelated to the patch?